bin/inx-chronicle/src/api/stardust/analytics/routes.rs

```diff
@@ -38,6 +38,8 @@ pub fn routes() -> Router {
             "/activity",
             Router::new()
                 .route("/addresses", get(address_analytics))
+                .route("/native-tokens", get(foundry_output_analytics))
+                .route("/nfts", get(nft_output_analytics))
```

Comment:
Please rename the methods in `get(...)` to `*-activity-analytics`. We should do this analogous for the `*-ledger-analytics`.